### PR TITLE
DM-11299 Coadded images displayed ignoring cutout size

### PR DESCRIPTION
**Pull request by tgoldina**
https://jira.lsstcorp.org/browse/DM-11299

This is continuation of Cindy's work on WISE cutouts
- when subsize is present in request, we assume that the image is cutout for both WISE and LSST.
- When cutouts are requested for LSST deepCoadd images, I_16 of [ImageServ API](https://confluence.lsstcorp.org/pages/viewpage.action?pageId=54854064) is called.
Unfortunately, the code should be fixed on DAX side. At the moment we get HTTP 404 Not Found for band 'r' and HTTP 502 Bad Gateway fro other bands. Test target 9.6 -1.1 

Known issues:
- I_16 of [ImageServ API](https://confluence.lsstcorp.org/pages/viewpage.action?pageId=54854064) produces error (LDSS SDSS deepCoadd image cutouts)
- Spinning wheel on table fetch error (addressed in another [pr-422](https://github.com/Caltech-IPAC/firefly/pull/422))
- Invalid cutout size (while being visually distinct) does not stop search. The issues with SizeInputField validation are addressed by DM-11386
- When searching WISE Images for cryo (only 3 bands available) or post-cryo (only 2 bands available), we still show 4 image widgets, unavailable images with 'Failed- Not Found' message. Example: WISE Images 3-band cryo for 'm1'.

More details about the first item:

The call for LSST SDSS cutouts produces an error on the DAX side. To test try

`curl -v -o band_i 'http://lsst-qserv-dax01.ncsa.illinois.edu:5000/image/v0/deepCoadd/cutout?ra=9.6&dec=-1.1&filter=i&width=5.0&height=5.0'`

`curl -v -o band_r 'http://lsst-qserv-dax01.ncsa.illinois.edu:5000/image/v0/deepCoadd/cutout?ra=9.6&dec=-1.1&filter=r&width=5.0&height=5.0'`

The content of band_i and band_r is different. The last one contains 'Image Not Found'. Also, I can see `HTTP/1.1 502 Bad Gateway` in the headers of the first request and `HTTP/1.1 404 NOT FOUND` in the headers of the second.

DM team is working to find a resolution of this issue

### CODE REVIEW

File: src/firefly/js/metaConvert/WiseRequestList.js [hunk 1]
```diff
@@ -32,6 +34,19 @@ export function makeWisePlotRequest(table, row, includeSingle, includeStandard,
     const builder = (plotId, reqKey, title, rowNum, extraParams) => {
         const req= svcBuilder(plotId, reqKey, title, rowNum, extraParams);
         req.setPreferenceColorKey('wise-color-pref');
+        const subsize = get(table, 'request.subsize');
+        if (subsize) {
```
Comment (cwang2016) [inline src/firefly/js/metaConvert/WiseRequestList.js]: what if 'subsize == 0', should that be also considered as 'cutout' ?

Comment (cwang2016) [inline src/firefly/js/metaConvert/WiseRequestList.js]: it is good that UI doesn't allow '0' entered. 
so this is not a concern.

**Comment by xiuqin** [comment]
I think subside should always  be >0.0. 
Field validation should check that too.

**Comment by cwang2016** [comment]
Update the field validation for 'Return Image Size' for LSST image search.

**Comment by cwang2016** [comment]
some finding, 
SizeInputField component needs some fixing to do the validation for all possible initial unit setting.

**Comment by tgoldina** [comment]
The last commit has 
- a fix the the cutout size of the WISE SDSS timeseries images
- workaround for DM-11386 (ignore invalid cutout size)
- (from Cindy) nullAllowed for size fields in CatalogSearchMethodType.jsx and LSSTImageSpatialType.jsx

**Pull request merged**